libr/core/cmd_open.c

```diff
@@ -511,7 +512,7 @@ static void cmd_omf (RCore *core, const char *input) {
 		// change perms of current map
 		int flags = (arg && *arg)? r_str_rwx (arg): 7;
 		ls_foreach (core->io->maps, iter, map) {
-			if (R_BETWEEN (map->from, core->offset, map->to)) {
+			if (R_BETWEEN (map->from, core->offset, map->to - 1)) {
```

Comment:
scary, is rbetween open-closed ranged? :P if we want to cover from 0 to UT64_MAX...

Reply:
`R_BETWEEN` is closed-interval. I think a counterpart for half-closed may be needed or fix all use sites of `R_BETWEEN`.          `core->offset <= map->to - 1` to make it work for address space around.     it could be simply   `core->offset < map->to` .